bin/generator-utils/templates.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+# shellcheck source=/dev/null
+source ./bin/generator-utils/utils.sh
+
+function create_test_file_template() {
+    local exercise_dir=$1
+    local slug=$2
+    local test_file="${exercise_dir}/tests/${slug}.rs"
+
+    cat <<EOT >"$test_file"
+use $(dash_to_underscore "$slug")::*;
+// Add tests here
+
+EOT
+
+    # fetch canonical_data
+    canonical_json=$(curl https://raw.githubusercontent.com/exercism/problem-specifications/main/exercises/"$slug"/canonical-data.json)
```

Comment:
You could consider adding some curl options to make this more robust (e.g. retrying).
See https://github.com/exercism/gleam/blob/main/bin/fetch-configlet#L9-L15 for some options.